### ISSUE TITLE
create solved environment export file without actually installing the environment

### ISSUE THREAD

**Issue by CagtayFabry** [mamba-org/mamba#2647]
Hi,
I am looking for a way to convert a small initial `environment.yaml` configuration file into a "fully solved" environment configuration as given by `mamba env export` but without downloading and installing the packages in between.

I am currently doing this to get the full environment configuration:
```bash
mamba env create -n demo_env -f environment.yaml
mamba env export -n demo_env -f environment_full.yaml --no-builds
mamba env remove -n demo_env
```

This gives the desired `environment_full.yaml` file but of course the initial step will install all packages.
I have tried to look through the mamba API for a way to access only the solver results but could not yet figure out a way.

Is there a way to achieve this with the existing `mamba` components?

**Comment by jonashaag** [comment]
Same as https://github.com/mamba-org/mamba/issues/2054?

**Comment by jonashaag** [comment]
Are you aware of [Conda-lock?](https://github.com/conda/conda-lock) Maybe you want to use that instead

**Comment by CagtayFabry** [comment]
> Same as #2054?

yes, I think we are looking for the same feature here. I seem to have missed your issue going through the list.
I will take a look at `conda-lock` as well.

closing as duplicate here
